src/main/java/com/lecturefeed/utils/TokenUtils.java

```diff
@@ -43,7 +53,9 @@ public static String getTokenValue(CustomAuthenticationService customAuthenticat
     }
 
     public static boolean isValidAdminToken(CustomAuthenticationService customAuthenticationService, TokenModel token){
-        return UserRole.ADMINISTRATOR.getRole().equals(TokenUtils.getTokenValue(customAuthenticationService,"role",token));
+        boolean isAdmin = UserRole.ADMINISTRATOR.getRole().equals(TokenUtils.getTokenValue(customAuthenticationService,"role",token));
+        boolean isNotExpired = System.currentTimeMillis() < Long.parseLong(getTokenValue(customAuthenticationService,"expirationDate",token));
```

Comment:
<=
would not make a big difference, but makes sence to me since it is the expiration date and should still be valid until the moment has passed